module/zfs/vdev.c

```diff
@@ -3775,5 +4112,13 @@ module_param(metaslabs_per_vdev, int, 0644);
 MODULE_PARM_DESC(metaslabs_per_vdev,
 	"Divide added vdev into approximately (but no more than) this number "
 	"of metaslabs");
+
+module_param(zfs_trim_mem_lim_fact, int, 0644);
+MODULE_PARM_DESC(metaslabs_per_vdev, "Maximum percentage of physical memory "
```

Comment:
This looks like a copy and paste error. Should be MODULE_PARM_DESC(zfs_trim_mem_lim_fact, ...